src/cljs/replumb/load.cljs

```diff
@@ -68,6 +105,15 @@
         src-path (file-paths src-paths file-path-without-ext)]
     (str src-path "." extension)))
 
+(defn cache-file-paths-for-load-fn
+  "Produces a sequence of file names to try reading for evaluation caching."
+  [cache-paths macros file-path-without-ext]
+  ;; try both eg. clojure/set and clojure_SLASH_set
+  (for [src-path (into (file-paths cache-paths file-path-without-ext)
+                       (file-paths cache-paths
+                                   (cache/cache-prefix-for-path file-path-without-ext macros)))]
+    [(str src-path ".js" ) (str src-path ".cache.json")]))
```

Comment:
Good job, very nice, it is too bad that `read-files-from-cache-and-callback!` has to know that we are passing a pair...should be transparent...but I don't think we have alternatives...